### DISCUSSION TITLE
I cant recorevery my second github account

### DISCUSSION THREAD

**Discussion by glysns** [community/community#23129] [New to GitHub]
<p>I forget my second github account password and i dont remember the  recovery e-mail,<br>
How can i get the account e-mail to try recovery my passoword ?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there - the best route to go would be filing a ticket with our Support team <a href="https://support.github.com/contact">here</a> as they are the best ones to assist in account-specific situations like this one :+1:t3:</p>
<p>Best of luck!</p>

**Comment by safsafiat76** [comment]
Hello ! My problem is that I can' t  find my second e-mail : [REDACTED DUE TO PII]
How can I find this second e-mail [REDACTED DUE TO PII] the recovery e-mail is [REDACTED DUE TO PII]
Please need help without phone number because my phone number [REDACTED DUE TO PII] and my device phone Samsung Galaxy S8 stolen by some peoples home attaque , everyday they continue destroy all my search or my emails all communication I do .

Now my only phone is Infinix x657B
Please help to unblock my second e-mail adress for my courses or research tools inside. Please

**Reply by queenofcorgis** [reply]
Hi @safsafiat76 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the community is unable to assist in these matters :)

**Comment by sina1100** [comment]
hi

**Reply by akasyaotomotiv** [reply]
Hello
